framework/test/jest/unit/specs/modules/chain/blocks/blocks.spec.js

```diff
@@ -1703,12 +1704,113 @@ describe('blocks', () => {
 		});
 	});
 
-	describe('loadBlocksDataWs', () => {
-		it.todo('a');
-	});
+	describe('loadBlocksFromLastBlockId', () => {
+		describe('when called without lastBlockId', () => {
+			it('should reject with error', async () => {
+				expect.assertions(1);
+				try {
+					await blocksInstance.loadBlocksFromLastBlockId();
+				} catch (err) {
+					expect(err.message).toBe('lastBlockId needs to be specified');
+				}
+			});
+		});
+
+		describe('when called without limit', () => {
+			const validLastBlock = {
+				height: 100,
+				id: 'block-id',
+			};
+
+			beforeEach(async () => {
+				stubs.dependencies.storage.entities.Block.get.mockResolvedValue([
+					validLastBlock,
+				]);
+			});
+
+			it('should use limit 1 as default', async () => {
```

Comment:
This is a confusing test. We say `limit 1` however I don't see any reference towards checking if it got called with `limit 1`.

Reply:
If the storage layer replaces an undefined limit to 1, pls add a comment to clarify this.

Reply:
in the `blocks. loadBlocksFromLastBlockId`, limit is set to `1` as default params